CONTRIBUTING.md

```diff
@@ -31,106 +41,112 @@ To send us a pull request, please:
 
 1. Fork the repository.
 2. Modify the source; please focus on the specific change you are contributing. If you also reformat all the code, it will be hard for us to focus on your change.
-3. Ensure local tests pass.
+3. Ensure local tests pass (check the [scripts](https://github.com/aws-samples/aws-cdk-examples/tree/master/scripts) directory)
 4. Commit to your fork using clear commit messages.
 5. Send us a pull request, answering any default questions in the pull request interface.
 6. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
 
 GitHub provides additional document on [forking a repository](https://help.github.com/articles/fork-a-repo/) and
 [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
 
-## Adding an example
+## Adding an Example <a name="New"></a>
 
-Create a directory with a name that is descriptive of the resource type or workflow you are trying to create, e.g. ecs-load-balanced-service. You will need to create the following four files in your example:
-1. `index.ts`, containing your CDK application code.
-2. `cdk.json`, containing:
+When adding a new example, there are several things important to consider as you implement (please create a `feature-request` issue to discuss these before writing your contribution):
```

Comment:
Might include a link to create the issue: https://github.com/aws-samples/aws-cdk-examples/issues/new?labels=feature-request%2C+needs-triage&template=feature-request.md

Reply:
```suggestion
When adding a new example, there are several things important to consider as you implement (please create a [`feature-request` issue](https://github.com/aws-samples/aws-cdk-examples/issues/new?labels=feature-request%2C+needs-triage&template=feature-request.md) to discuss these before writing your contribution):
```